js/script.js

```diff
@@ -59,6 +82,26 @@
     }
   })
 
+  app.service("UserStorage", function($http, API){
+
+    this.user = null;
+
+    this.setUser = function(){
+      var self = this
+      $http.get(API.BASE + API.AUTH + "info/")
+      .then(function(response){
+        if (response.data !== "AnonymousUser"){
+          self.user = response.data
+        }
+      })
+    }
+
+    this.clearUser = function(){
+      this.user = null
+    }
```

Comment:
I think here must be another method - `getUser`, like getter. It will check if `user` is null then request it from server.